service/controller/orgcluster/resource/orgclusterresources/create.go

```diff
@@ -0,0 +1,185 @@
+package orgclusterresources
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/giantswarm/microerror"
+	rbacv1 "k8s.io/api/rbac/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	pkgkey "github.com/giantswarm/rbac-operator/pkg/key"
+	"github.com/giantswarm/rbac-operator/pkg/label"
+	"github.com/giantswarm/rbac-operator/pkg/project"
+	"github.com/giantswarm/rbac-operator/service/controller/orgcluster/key"
+)
+
+// Ensures that
+// - Roles for read/write access to org cluster resources are ensured in each cluster namespace
+// - For each RoleBinding in an org-namespace that references the read/write org cluster resource clusterRole,
+//   RoleBindings are created in the organizations cluster namespaces which reference above Role
+func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
+	var err error
+
+	cl, err := key.ToCluster(obj)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	orgNamespace := cl.Namespace
+	if !pkgkey.IsOrgNamespace(orgNamespace) {
+		return nil
+	}
+
+	// List roleBindings in org-namespace
+	r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("Listing RoleBindings in namespace %s.", orgNamespace))
+	orgRoleBindings, err := r.k8sClient.K8sClient().RbacV1().RoleBindings(orgNamespace).List(ctx, metav1.ListOptions{})
+	if err != nil {
+		return microerror.Mask(err)
+	} else if len(orgRoleBindings.Items) == 0 {
+		r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("No RoleBindings found in namespace %s.", orgNamespace))
+		return nil
+	}
+
+	for _, referencedRole := range referencedClusterRoles() {
```

Comment:
This loop will run for each role we have defined in the `config.go` file. Right now it's just the read and write cluster apps roles. For each of them we check which subjects need access and then ensure resources.